particles/Processing/ImageToTensor.particle

```diff
@@ -0,0 +1,7 @@
+import './schemas/Image.schema'
+import './schemas/MlResource.schema'
+
+particle ImageToTensor in './js/ImageToTensor.js'
+  in Image {(Text or URL) url} image
+  out TfTensor imageTensor
+  //description `Converts an Image URL to a Tensor3D`
```

Comment:
I like the flat folder structure + the idea that the name of the file matches the particle.